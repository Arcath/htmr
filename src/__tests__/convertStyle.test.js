/* eslint-env jest */
import convertStyle from '../convertStyle';

test('basic conversion', () => {
  const style = 'display: none; margin: 10px';
  expect(convertStyle(style)).toMatchSnapshot();
});

test('vendor prefixes', () => {
  const style = '-ms-text-size-adjust: 100%; -webkit-text-size-adjust: 100%';
  expect(convertStyle(style)).toMatchSnapshot();
});

test('html entities', () => {
  const style =
    'font-family: Consolas, &quot;Liberation Mono&quot;, Menlo, Courier, monospace';
  expect(convertStyle(style)).toMatchSnapshot();
});
