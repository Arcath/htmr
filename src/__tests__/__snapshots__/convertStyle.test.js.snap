// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`basic conversion 1`] = `
Object {
  "display": "none",
  "margin": "10",
  "padding": "0 10px",
}
`;

exports[`html entities 1`] = `
Object {
  "fontFamily": "Consolas, \\"Liberation Mono\\", Menlo, Courier, monospace",
}
`;

exports[`vendor prefixes 1`] = `
Object {
  "WebkitTextSizeAdjust": "100%",
  "msTextSizeAdjust": "100%",
}
`;
